management/user_test.go

```diff
@@ -260,6 +260,37 @@ func TestUser(t *testing.T) {
 		}
 		t.Logf("%v\n", us)
 	})
+
+	t.Run("Link", func(t *testing.T) {
+		ulAlice, err := m.User.Search(Query(`email:"alice@example.com"`))
```

Comment:
We can also create a couple users, that better represent linking between two identities. We can make a better example of linking Batman to Bruce Wayne or Heisenberg to Walter White 😄 

Just remember to clean up once done

```go
t.Cleanup(func() { 
	m.User.Delete(walter.GetID())
	m.User.Delete(heisenberg.GetID())
})
```